frontend/src/routes/_layout/inquiries.tsx

```diff
@@ -1,27 +1,98 @@
 import {
   Container,
   Heading,
-} from "@chakra-ui/react"
-import { createFileRoute} from "@tanstack/react-router"
-import { z } from "zod"
+  SkeletonText,
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { useQuery } from "@tanstack/react-query";
+import { createFileRoute } from "@tanstack/react-router";
+import { z } from "zod";
 
-import Navbar from "../../components/Common/Navbar"
-import AddInquiry from "../../components/Inquiries/AddInquiry"
+import * as InquiriesService from "../../client/services/inquiriesService";
+import Navbar from "../../components/Common/Navbar";
+import AddInquiry from "../../components/Inquiries/AddInquiry";
 
 // Already typed by zod library https://zod.dev/
 // eslint-disable-next-line
 const inquiriesSearchSchema = z.object({
-// eslint-disable-next-line
+  // eslint-disable-next-line
   page: z.number().catch(1),
-})
+});
 
 // createFileRoute is already typed by tanstack-router: https://tanstack.com/router/latest/docs/framework/react/guide/type-safety
 // eslint-disable-next-line
 export const Route = createFileRoute("/_layout/inquiries")({
   component: Inquiries,
-    // eslint-disable-next-line
+  // eslint-disable-next-line
   validateSearch: (search) => inquiriesSearchSchema.parse(search),
-})
+});
+
+function getInquiriesQueryOptions() {
+  return {
+    queryKey: ["inquiries"],
+    queryFn: () => InquiriesService.readInquiries(),
+  };
+}
+
+function InquiriesTable() {
+  const { data: inquiries, isPending } = useQuery({
+    ...getInquiriesQueryOptions(),
+  });
+
+  const sortedInquiries = inquiries?.data.sort((a, b) => {
+    const dateA = new Date(a.created_at).getTime();
+    const dateB = new Date(b.created_at).getTime();
+    return dateB - dateA; // Newest to oldest
+  });
+
+  return (
+    <>
+      <TableContainer>
+        <Table>
+          <Thead>
+            <Tr>
+              <Th>ID</Th>
+              <Th>Text</Th>
+              <Th>Created At</Th>
+            </Tr>
+          </Thead>
+          {isPending ? (
+            <Tbody>
+              <Tr>
+                {new Array(3).fill(null).map((_, index) => (
+                  <Td key={index}>
+                    <SkeletonText noOfLines={1} />
+                  </Td>
+                ))}
+              </Tr>
+            </Tbody>
+          ) : (
+            <Tbody>
+              {sortedInquiries?.map((inquiry) => (
+                <Tr
+                  key={inquiry.id}
+                  onClick={() => {
+                    console.log(inquiry);
+                  }}
+                >
+                  <Td>{inquiry.id}</Td>
+                  <Td>{inquiry.text}</Td>
+                  <Td>{new Date(inquiry.created_at).toISOString()}</Td>
```

Comment:
Here is where you can use day.js to format the date better.  Do the formatting in the code above the return statement, not in the JSX

Reply:
looks great!